presto-main/pom.xml

```diff
@@ -468,6 +468,47 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.facebook.drift</groupId>
+                <artifactId>drift-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>execution1</id>
+                        <goals>
+                            <goal>generate-thrift-idl</goal>
+                        </goals>
+                        <phase>process-classes</phase>
+                        <configuration>
+                            <classes>
+                                <class>com.facebook.presto.execution.TaskStatus</class>
+                            </classes>
+                            <recursive>true</recursive>
+                            <outputFile>target/presto-internal-${project.version}.thrift</outputFile>
```

Comment:
how about keep the file in a new folder instead of creating it under the parent folder?

Reply:
Moved to a new module as suggested originally.